src/components/signoff/SignoffToolBar.tsx

```diff
@@ -110,10 +110,27 @@ export default function SignoffToolBar({
   const { status } = source;
 
   const canRequestReview = canEdit && isEditor(source, sessionState);
+
+  let toReviewEnabled =
+    sessionState.serverInfo?.capabilities?.signer?.to_review_enabled === true;
+  if (toReviewEnabled) {
+    const resourceMatch =
+      sessionState.serverInfo?.capabilities?.signer?.resources?.find(
+        x =>
+          x.source.bucket === source.bid &&
+          x.source.collection === source.cid &&
+          x.destination.bucket === destination.bid &&
+          x.destination.collection === destination.cid
+      );
+    toReviewEnabled =
+      !resourceMatch || resourceMatch.to_review_enabled !== false;
```

Comment:
I think it would be clearer to write:
```js
if (resourceMatch && resourceMatch.hasOwnProperty(to_review_enabled)) {
    toReviewEnabled = resourceMatch.to_review_enabled;
}
```

Reply:
Ended up refactoring this a bit to be easier to read.